DigitalLearningSolutions.Web.Tests/ViewModels/Support/SearchableFaqTests.cs

```diff
@@ -5,21 +5,20 @@
     using FluentAssertions;
     using NUnit.Framework;
 
-    public class FaqViewModelTests
+    public class SearchableFaqTests
     {
         [Test]
-        public void FaqViewModelTests_returns_model_with_cleaned_searchName()
+        public void SearchableFaq_returns_model_with_cleaned_SearchableFaqAnswer()
         {
             // Given
             var faq = FaqTestHelper.GetDefaultFaq();
-            var expectedSearchableName = "A common question?  p Helpful content  p ";
+            var expectedSearchableFaqAnswer = " p Helpful content  p ";

```

Comment:
Sorry, this is one of those "it was just a refactor!" cases, but this test could be improved by specifying the answer in line 14 and inlining the expected answer in `Then`. No need for re-review.

Reply:
done.